codegen/java-gen/src/main/scala/kalix/codegen/java/ActionTestKitGenerator.scala

```diff
@@ -141,9 +143,9 @@ object ActionTestKitGenerator {
   }
 
   def generateServices(service: ModelBuilder.ActionService): String = {
-    require(!service.commands.isEmpty, "empty `commands` not allowed")
```

Comment:
I don't know why we had this `require` in the testkit for Java? It's not in the testkit for Scala.
With this additional filter of ignore methods it can be possible to mark all methods as ignore (even though it doesn't make sense).